requirements.yaml.m4

```diff
@@ -17,5 +17,6 @@
       prefix: piksi_fpga/NAP_VERSION
       files:
         - piksi_prod_fpga.bit
+        - piksi_sdk_fpga.bit
```

Comment:
Based on discussion with @silverjam, this seems un-necessary, I'll drop it in the update